build/AzurePipelinesTemplates/MUX-BuildProject-Steps.yml

```diff
@@ -38,7 +48,7 @@ steps:
       vsVersion: 15.0
       platform: '$(buildPlatform)'
       configuration: '$(buildConfiguration)'
-      msbuildArgs: '/p:AppxPackageDir="${{ parameters.appxPackageDir }}" /p:AppxBundle=Never /p:AppxSymbolPackageEnabled=false /binaryLogger:$(Build.SourcesDirectory)/${{ parameters.solutionPath }}.$(buildPlatform).$(buildConfiguration).binlog -property:MUXVersionBuild=$(builddate_yymm) -property:MUXVersionRevision=$(builddate_dd)$(buildrevision)'
+      msbuildArgs: '/p:AppxPackageDir="${{ parameters.appxPackageDir }}" /p:AppxBundle=Never /p:AppxSymbolPackageEnabled=false /binaryLogger:$(Build.SourcesDirectory)/${{ parameters.solutionPath }}.$(buildPlatform).$(buildConfiguration).binlog -property:MUXVersionBuild=$(builddate_yymm) -property:MUXVersionRevision=$(builddate_dd)$(buildrevision) /p:VCToolsInstallDir="$(VCToolsInstallDir)\" -p:PGOBuildMode=$(PGOBuildMode)'
```

Comment:
Is is possible to split this line across multiple lines in yml? I don't know if yml supports that, but it would aid readability. This line is becoming a beast!